docs/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml

```diff
@@ -3185,6 +3177,155 @@ ENDMACRO;
 
         <para>There is also a right-click pop-up menu with the all the options
         found on the <emphasis role="bold">Edit</emphasis> menu.</para>
+
+        <sect3 id="IDE_Support_For_Other_Languages">
+          <title>Support for Other Languages</title>
+
+          <para>In addition to providing native support for ECL Files, the ECL
+          IDE also provides support for working with other languages. The IDE
+          provides colored file type differentiation, where different file
+          types all display in a different color. The IDE also provides
+          colorized keywords in the supported languages. The colorization is
+          set up as a default, but can be customized. Other languages that the
+          IDE supports</para>
+
+          <para><informaltable colsep="1" frame="all" rowsep="1">
+              <tgroup cols="2">
+                <colspec align="left" colwidth="122.40pt" />
+
+                <tbody>
+                  <row>
+                    <entry><emphasis role="bold">Language</emphasis></entry>
+
+                    <entry><emphasis
+                    role="bold">Functionality</emphasis></entry>
+                  </row>
+
+                  <row>
+                    <entry>ECL</entry>
+
+                    <entry><itemizedlist>
+                        <listitem>
+                          <para>Colorized file differentiation</para>
```

Comment:
Colorized file differentiation is confusing. What you are trying to say is: 
Repository icon is colorized to identify the file type.